stable/presto/templates/deployment-worker.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if gt .Values.server.workers 0.0 }}
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "presto.worker" . }}
+  labels:
+    app: {{ template "presto.name" . }}
+    chart: {{ template "presto.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: worker
+spec:
+  replicas: {{ .Values.server.workers }}
+  selector:
+    matchLabels:
+      app: {{ template "presto.name" . }}
+      release: {{ .Release.Name }}
+      component: worker
+  template:
+    metadata:
+      labels:
+        app: {{ template "presto.name" . }}
+        release: {{ .Release.Name }}
+        component: worker
+    spec:
+      volumes:
+        - name: config-volume
+          configMap:
+            name: {{ template "presto.worker" . }}
+      containers:
+        - name: {{ .Chart.Name }}-worker
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/bash"]
+          args:
+            - {{ .Values.server.config.path }}/docker-presto.sh
+          volumeMounts:
+            - mountPath: {{ .Values.server.config.path }}
+              name: config-volume
+          livenessProbe:
+            exec:
+              command:
```

Comment:
Use consistent list indentation. To match the prevailing style, indent one more level. Apply everywhere.